x-pack/plugins/session_view/public/components/session_view/hooks.ts

```diff
@@ -74,6 +78,31 @@ export const useFetchSessionViewProcessEvents = (
   return query;
 };
 
+export const useFetchSessionViewAlerts = (sessionEntityId: string) => {
+  const { http } = useKibana<CoreStart>().services;
+  const query = useQuery(
+    'sessionViewAlerts',
```

Comment:
I think it might be good to store our use query keys as constant on a common folder, so we avoid duplication